src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/caseworker/event/page/GeneralDirectionOrders.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.adoption.adoptioncase.caseworker.event.page;
+
+
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.CaseData;
+import uk.gov.hmcts.reform.adoption.adoptioncase.model.DirectionsOrderData;
+import uk.gov.hmcts.reform.adoption.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.reform.adoption.common.ccd.PageBuilder;
+
+public class GeneralDirectionOrders implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder.page("manageOrderGD1")
+            .showCondition("manageOrderType=\"generalDirectionsOrder\"")
+            .pageLabel("General directions order")
+            .label("generalDirectionOrderLabel2", "## General directions order")
+            .label("generalDirectionOrderLabel1", "## Enter hearing details")
+            .complex(CaseData::getDirectionsOrderData)
+            .label("generalDirectionOrderBy", "### Order by", null, true)
+            .mandatory(DirectionsOrderData::getOrderBy)
+            .label("generalDirectionDateOfOrderMade", "### Date order made", null, true)
+            .mandatory(DirectionsOrderData::getDateOfGeneralDirectionOrderMade)
+            .label("generalDirectionPreamble", "### Preamble", null, true)
+            .optional(DirectionsOrderData::getGeneralDirectionOrderPreamble)
+            .label("generalDirectionBodyOfOrder", "### Body of the order", null, true)
+            .optional(DirectionsOrderData::getGeneralDirectionBodyOfTheOrder)
+            .label("generalDirectionCostOrders", "### Cost orders", null, true)
+            .optional(DirectionsOrderData::getGeneralDirectionCostOfOrder)
+            .label("generalDirectionIncludedInTheOrder", "### Directions to include in the order",
+                   null, true
+            )
+            .optional(DirectionsOrderData::getGeneraDirectionsIncluded)
+            .label(
+                "generalDirectionOrderTypeYouNeedToSend",
+                "### Is there another order type you need to send instead",
+                null,
+                true
+            )
+            .mandatory(DirectionsOrderData::getIsThereAnyOrderTypeYouNeedToSend)
+            .mandatory(DirectionsOrderData::getGeneralDirectionOrderTypes, "isThereAnyOrderTypeYouNeedToSend=\"Yes\"")
+            .done()
+            .page("manageOrderGD2")
+            .showCondition(
+                "manageOrderType=\"generalDirectionsOrder\" AND generalDirectionOrderTypes=\"generalDirectionProductionOrder\"")
+            .label("manageOrderGdGov", "<p>To the governor<br>"
```

Comment:
Label should be with all initial letter capitalised. 'To The Governor'

Reply:
corrected